lms/static/js/utils/dropdown.js

```diff
@@ -0,0 +1,203 @@
+var edx = edx || {},
+
+    Dropdown = (function() {
+
+        var _dropdown = {
+
+            opts: {
+                page: $(document),
+                midpoint: ($(window).width() / 2),
+                button: $('.button-more.has-dropdown'),
+                button_active: 'is-active',
+                menu: $('.dropdown-menu'),
+                menu_inactive: 'is-hidden',
+                menu_active: 'is-visible',
+                menu_align: 'align-',
+                text_align: 'control-text-align'
+            },
+
+            init: function(parent) {
+
+                if (_dropdown.opts.button.length && _dropdown.opts.menu.length) {
+
+                    if (parent) {
+                        _dropdown.opts.page = $(parent);
+                    }
+
+                    _dropdown.listenForClick();
+                    _dropdown.listenForKeypress();
+                }
+            },
+
+            listenForClick: function() {
+
+                _dropdown.opts.button.on('click', function() {
+                    _dropdown.closeDropdownMenu(); // close any open menus
```

Comment:
Should this be "closeDropdownMenus" instead of "closeDropdownMenu"? Your comment says "close any open menus".